draft/aeap-mvp.rst

```diff
@@ -0,0 +1,33 @@
+AEAP MVP
+========
+
+Changes to the UIs
+------------------
+
+- The secondary self addresses (see below) are shown in the UI, but not editable.
+
+That's all!
```

Comment:
not sure that's through.  When "changing an e-mail address" we probably need to ask back if it should be done with "forwarding" porting logic or not -- and adding the old as a secondary e-mail address depending on the answer.

Reply:
I don't actually think anyone would want to press "no" in this dialog, as it will break all groups you're in and I can't come up with any advantages over pressing "yes"

Reply:
But it probably makes sense to show an informational dialog